app/src/main/java/com/dicoding/picodiploma/sejiwaproject/features/match/detail/DetailMatchActivity.kt

```diff
@@ -59,8 +73,17 @@ class DetailMatchActivity : AppCompatActivity(),
             away_red.invisible()
             home_yellow.invisible()
             away_yellow.invisible()
+            date_match.invisible()
```

Comment:
Should be write like last our discussion.

Reply:
Resolved, thanks